watcher.js

```diff
@@ -1,80 +0,0 @@
-/*
-    Copyright (c) 2016 eyeOS
-
-    This file is part of Open365.
-
-    Open365 is free software: you can redistribute it and/or modify
-    it under the terms of the GNU Affero General Public License as
-    published by the Free Software Foundation, either version 3 of the
-    License, or (at your option) any later version.
-
-    This program is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU Affero General Public License for more details.
-
-    You should have received a copy of the GNU Affero General Public License
-    along with this program. If not, see <http://www.gnu.org/licenses/>.
-*/
-
-var fs = require('fs');
-var exec = require('child_process').exec;
-
-function reloadDnsmasq(){
-	exec('pidof dnsmasq | xargs --no-run-if-empty kill -1', function (error, stdout, stderr) {
-		console.log("Restarting dnsmasq.");
-		console.log('Error: ', error);
-		console.log('stdout: ', stdout);
-		console.log('stderr: ', stderr);
-	});
-};
-
-fs.watchFile('/hosts/hosts.serf', function () {
-	console.log("Watching hosts.serf");
-	reloadDnsmasq();
-});
-
-fs.watchFile('/hosts/hosts.static', function () {
```

Comment:
This file is being removed, however eyeos-run-server does not watch these files. I'm a little uncomfortable merging this even though it wasn't being used (That might have been an accident).

Could someone please investigate why this wasn't being used?